client/app/(authenticated)/bceidbusiness/industry_user/dashboard/operations/[operation]/[formSection]/page.tsx

```diff
@@ -0,0 +1,10 @@
+// 🚩 flagging that for shared routes between roles, "Page" code is a component for code maintainability
```

Comment:
Non-admin industry users should be able to access operations and it looks like the pages weren't in the app yet, so I added here